src/bindings/c/tests/test_model_repo.hpp

```diff
@@ -2,50 +2,46 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 #include <fstream>
+#include <random>
+
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+#include "openvino/pass/manager.hpp"
 
 namespace TestDataHelpers {
 
-static const char kPathSeparator =
-#if defined _WIN32 || defined __CYGWIN__
-    '\\';
-#else
-    '/';
-#endif
+static const std::string model_bin_name = "test_model.bin";
+static const std::string model_xml_name = "test_model.xml";
+static const std::string model_exported_name = "test_exported_model.blob";
 
-inline std::string getModelPathNonFatal() noexcept {
-    if (const auto envVar = std::getenv("MODELS_PATH")) {
-        return envVar;
-    }
-
-#ifdef MODELS_PATH
-    return MODELS_PATH;
-#else
-    return "";
-#endif
+inline void generate_test_model() {
+    ov::pass::Manager manager;
+    manager.register_pass<ov::pass::Serialize>(model_xml_name, model_bin_name);
+    auto function = ngraph::builder::subgraph::makeConvPoolReluNoReshapes({1, 3, 227, 227});
+    manager.run_passes(function);
 }
 
-inline std::string get_models_path() {
-    return getModelPathNonFatal() + kPathSeparator + std::string("models");
-};
-
-inline std::string get_data_path() {
-    if (const auto envVar = std::getenv("DATA_PATH")) {
-        return envVar;
-    }
+inline std::string get_model_xml_file_name() {
+    generate_test_model();
```

Comment:
Seems like the generated IR files need to be remove

Reply:
good catch. Suggest their removal in tests's TearDown function

Reply:
done